LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Can we just straight out copy the LICENSE file from JupyterLab? It has some subtle differences, including the Copyright statement, and no "BSD 2-Clause License" line.